interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: This in JavaScript is something you can use to call on an object within a react component.
 
-  Researched answer:
+  Researched answer: The 'this' keyword refers to an object. The object that it refers to depends on which object is being called. It's usually used with a period after it then the name of the object.
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: A dynamic interface that can be used on websites to edit the webpage in realtime without having to refresh the page.
 
-  Researched answer:
+  Researched answer: React is a JavaScript library for building user interfaces. It's used to make websites dynamic and to change things on a webpage automatically without having to refresh.
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: Not sure of the top of my head because we didn't go to deep into this but I will research this answer and elaborate below.
 
-  Researched answer:
+  Researched answer: Render is the only required method in a react class component.
```

Comment:
Great!  I appreciate you answering even if you didn't know without your notes.  Oftentimes an interviewer will keep asking questions until you don't know and want to see how you respond in that situation.